src/app/clusters/energy-evse-server/energy-evse-server.h

```diff
@@ -38,13 +39,31 @@ namespace EnergyEvse {
 constexpr int64_t kMinimumChargeCurrent         = 0;
 constexpr int64_t kMaximumChargeCurrent         = 80000;
 constexpr uint32_t kMaxRandomizationDelayWindow = 86400;
+constexpr uint8_t kEvseTargetsMaxNumberOfDays   = 7;
+constexpr uint8_t kEvseTargetsMaxTargetsPerDay  = 10;
+
+struct EvseChargingTarget
+{
+    uint16_t targetTimeMinutesPastMidnight;
+    Optional<chip::Percent> targetSoC;
```

Comment:
This is inside `namespace chip`, no?  That applies to various places in this header.

Reply:
Sorry but not sure what is wrong here. Would you like `struct EvseChargingTarget` -> `struct->chip::EvseChargingTarget`?

Reply:
I would like `Optional<Percent> targetSoC;`